core/utils.c

```diff
@@ -549,6 +550,23 @@ lwm2m_server_t * utils_findBootstrapServer(lwm2m_context_t * contextP,
 #endif
 }
 
+#ifndef LWM2M_CLIENT_MODE
+lwm2m_client_t * utils_findClient(lwm2m_context_t * contextP,
+                                  void * fromSessionH)
+{
+    lwm2m_client_t * targetP;
+
+    targetP = contextP->clientList;
```

Comment:
Check contextP == NULL

Reply:
I think we had similar discussions in other cases and the argument beeing that in order to keep a minimal footprint we need to trust calling functions and callback to not do anything vile.